javascript/react-js/react-introduction.md

```diff
@@ -0,0 +1,136 @@
+### Introduction
+
+The last lesson has given you a quick overview about the three most popular Frontend Frameworks (Vue.js, Angular, React.js). This section aims to provide you with all you need to know about React, to feel the power of a Frontend Framework and also to help you create highly scalable React applications as well as beautiful websites. But before we jump right into it we need to answer one question.
+
+### Why React?
+
+As you already know from the previous section, React is one of the most powerful and widely used Frontend frameworks. If you have done the previous lesson and are here because you decided react is the framework to go for, you can skip this "Why React?" part of this lesson. However if you just stumbled over this, or if you are still unsure about weather to learn react or not, let me convince you.
+
+The landscape for frontend frameworks has been changing a lot over the last few year, so it is totaly understandable to be "scared" of going for the "wrong" one.
+[This](https://stackoverflow.blog/2018/01/11/brutal-lifecycle-javascript-frameworks/) article shows the recent development of frontend frameworks quite well.
+
+However, once you started diving deeper into a Framework you will start loving it, I promise. It makes your coding scalable, much more readable and probably a thousand times more efficient (just my modest estimation).
+
+Just to name a few reasons why to learn React.js:
+
+- Reusability of components
+- Amazing community
+- React is unopinionated, which means that it won't force you to follow any specific patterns or logics, it's all up to you.
+- Pretty easy to learn, especially when you already have a good grasp of JavaScript and Html, which you do when you followed the previous courses
+- React uses the virtual DOM, which makes it extremely fast and efficient
+
+If you need more convincing check out [this](https://medium.com/@SilentHackz/top-10-reasons-why-you-should-learn-react-right-now-f7b0add7ec0d) article and [this](https://medium.com/@cassiozen/10-reasons-why-every-developer-should-learn-react-87fbfef2cb91).
```

Comment:
Citing React's popularity via the [StackOverflow Developer Survey 2020](https://insights.stackoverflow.com/survey/2020#technology-web-frameworks-all-respondents2) - would prove very useful here and resonate with the reader a bit more than the addition of another top 10.

Reply:
Great survey, haven't seen it myself